### ISSUE TITLE
bad address mode with comma

### ISSUE THREAD

**Issue by ksherlock** [ksherlock/qasm#16]
```
Bad address mode in line: 1.
                     1              pea  #','         
```

**Comment by ksherlock** [comment]
```
 pea #','
 pea #'''
```

both accepted in merlin, cause problems in qasm.

**Comment by ksherlock** [comment]
Merlin complains about `pea #'''`, `asc '''`, etc as well

**Comment by ksherlock** [comment]
fixed 0fb0dc0cc123da14cca4547f96d0197bbf46bb33
